lib/Stats.js

```diff
@@ -436,9 +437,19 @@ class Stats {
 					}, Object.create(null))
 				};
 				if (showPerformance) {
-					obj.entrypoints[name].isOverSizeLimit = ep.isOverSizeLimit;
+					obj[name].isOverSizeLimit = cg.isOverSizeLimit;
 				}
 			}
+
+			return obj;
+		};
+
+		if (showEntrypoints) {
+			obj.entrypoints = fnChunkGroup(compilation.entrypoints);
+		}
+
+		if (showChunkGroups) {
+			obj.chunkGroups = fnChunkGroup(compilation.namedChunkGroups);
```

Comment:
Rename it to namedChunkGroups

Reply:
should the option name also change to `namedChunkGroups`?